AppCenter/AppCenterTests/MSACAppCenterTests.m

```diff
@@ -1065,4 +1065,13 @@ - (void)testDisableNetworkRequestsAfterStart {
   XCTAssertEqual(tokenObject, self.channelGroupMock);
 }
 
+- (void)testSetDataResidencyRegion {
+  // If
+  NSString *expectedDataResidencyRegion = @"rg";
+  [MSACAppCenter setDataResidencyRegion:expectedDataResidencyRegion];
+
+  // Then
+  XCTAssertEqualObjects(expectedDataResidencyRegion, [MSACAppCenter sharedInstance].dataResidencyRegion);
```

Comment:
I think you should use a static getter here instead of directly getting the property from the instance.

Reply:
Corrected